openpathsampling/pathmover.py

```diff
@@ -1503,12 +1502,6 @@ class RandomAllowedChoiceMover(RandomChoiceMover):
     from sub movers that actually can succeed because they have samples in all
     required input_ensembles
 
-    Attributes
-    ----------
-    movers : list of PathMover
-        the PathMovers to choose from
-    weights : list of floats
-        the relative weight of each PathMover (does not need to be normalized)
```

Comment:
help does not. `__doc__` is either inherited from super or the new is taken. Sphinx will do this if not turned off